src/workerd/jsg/limits.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <kj/common.h>
+
+namespace workerd::jsg {
+
+class Lock;
+
+class IsolateLimitEnforcer {
+public:
+  // Called when performing a cypto key derivation function (like pbkdf2) to determine if
```

Comment:
I think the function needs to be specific to PBKDF2. I don't think the same limit is likely to be appropriate for other KDFs that work entirely differently.